index2.html

```diff
@@ -0,0 +1,149 @@
+        <div><img class="counter" src="snapshotimages/Counter.png" alt="counter"></div>
+        <div><a href="http://">Back End Dev</a></div>
+        <div><img class="counter" src="snapshotimages/Counter.png" alt="counter"></div>
+        <div><a href="http://">2015</a></div>
+      </nav>
+      <div><p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
+      <ul class="languages">
+        <li><button class="language-button">html</button></li>
+        <li><button class="language-button">css</button></li>
+        <li><button class="language-button">javascript</button></li>
+      </ul>
+      <div><button class="see-project-button">See Project</button></div>
+    </div>
+  </div>
+  </section>
+
+  <section class="about-myself">
+    <header class="about-me-header">
+      <div><h2>About Myself</h2></div>
+    </header>
+    <div><p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></div>
+    <div class="social-media">
+      <div class="headline"><a class="social-media-link" href="http://"> LET’S CONNECT</a></div>
+      <ul class="social-icons">
+       <li class="icon-container"><a href="http://"><img src="icon-images/Icon -GitHub.png" alt="github icon"></a></li>
+       <li class="icon-container"><a href="http://"><img src="icon-images/Icon - Linkedin.png" alt="linkedin icon"></a></li>
+       <li class="icon-container"><a href="http://"><img src="icon-images/Icon - Angelist.png" alt="angelist icon"></a></li>
+       <li class="icon-container"><a href="http://"><img src="icon-images/Icon - Twitter.png" alt="tweeter icon"></a></li>
+       <li class="icon-container"><a href="http://"><img src="icon-images/Icon - Medium.png" alt="medium icon"></a></li>
+      </ul>
+      </div>
+      <div><button class="get-my-resume-button">Get My Resume</button></div>
+      <ul class="self-profession">
+        <li class="language-items">
+        <div class="language-select">
+          <h3>Languages</h3>
+          <img class="dropdown" src="icon-images/Icon - Chevron - Down - Larger.png" alt="dropdown icon">
+        </div>
+          <div class="java-html-css-container">
+            <div class="language-type-container">
+              <img class="language-images" src="language-icons/javascript.png" alt="javascript icon">
+              <h5>javascript</h5>
+            </div>
+            <div  class="language-type-container">
+              <img class="language-images" src="language-icons/html.png" alt="html icon">
+              <h5>HTML</h5>
+            </div>
+            <div  class="language-type-container">
+              <img class="language-images" src="language-icons/css.png" alt="css icon">
+              <h5>css</h5>
+            </div>
+            
+          </div>
```

Comment:
- [ ] This section is a list similar to the parent list, so using a `ul` tag with the `li` tags for the items would be a better approach. Kindly change that.